packages/wonder-blocks-dropdown/components/select-box.md

```diff
@@ -0,0 +1,80 @@
+### Example: Regular and light select box versions
+These select boxes also have specified widths.
+
+```js
+const {View} = require("@khanacademy/wonder-blocks-core");
+const {StyleSheet} = require("aphrodite");
+
+const styles = StyleSheet.create({
+    row: {
+        flexDirection: "row",
+    },
+    darkBackgroundWrapper: {
+        background: "black",
+        padding: 10,
+    },
+    strutLike: {
+        width: 8,
+    }
+});
+<View style={[styles.row]}>
+    <SelectBox
+        isPlaceholder={false}
+        onClick={() => console.log("regular selectbox selected")}
+        style={{width: 200}}
+    >
+        Regular selectbox
+    </SelectBox>
+    <View style={[styles.strutLike]} />
```

Comment:
Why doesn't `Strut` work here?  We could also set `marginRight` on the first `SelectBox`.